cds_ils/static_pages/search_guide.html

```diff
@@ -0,0 +1,674 @@
+<h2 id="cds-videos">CERN library catalogue</h2>
```

Comment:
```suggestion
<h2>CERN library catalogue</h2>
```

I think I have removed the extra title, it looks a bit weird maybe. Can you check the other static pages?